src/BookNavigator/volumes/volumes-provider.js

```diff
@@ -0,0 +1,71 @@
+import { html } from 'lit-element';
+
+import sortAscendingIcon from '../assets/icon_sort_ascending.js';
+import sortDescendingIcon from '../assets/icon_sort_descending.js';
+import volumesIcon from '../assets/icon_volumes.js';
+
+import './volumes.js';
+
+export default class VolumesProvider {
+
+  constructor(optionChange, baseHost, bookreader) {
```

Comment:
Minor, but may I suggest making the callback the last argument instead of the first. Looking at the instantiation of this above, the last two arguments get lost.